h2o-docs/src/product/data-science/word2vec.rst

```diff
@@ -0,0 +1,117 @@
+Word2vec
+--------
+
+Introduction
+~~~~~~~~~~~~
+
+In text-rich datasets, the Word2vec algorithm computes vector representations of words within the data and outputs those word vectors into a single file. The vector space can include hundreds of dimensions, with each unique word in the sample (`corpus <https://en.wikipedia.org/wiki/Corpus_linguistics>`__) being assigned a corresponding vector in the space. In addition, words that share similar contexts in the corpus are placed in close proximity to one another in the space.
+
+A Word2vec demo in R using a Craigslist job titles dataset available `here <https://github.com/h2oai/h2o-3/blob/master/h2o-r/demos/rdemo.word2vec.craigslistjobtitles.R>`__.
+
+**Note**: Word2vec is not currently supported under Python.
+
+Defining a Word2vec Model
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  `model_id <algo-params/model_id.html>`__: (Optional) Specify a custom name for the model to use as a reference. By default, H2O automatically generates a destination key.
+
+-  `training_frame <algo-params/training_frame.html>`__: (Required) Specify the dataset used to build the model. **NOTE**: In Flow, if you click the **Build a model** button from the ``Parse`` cell, the training frame is entered automatically.
+
+- **min_word_freq**: Specify an integer for the minimum word frequency. Word2vec will discard words that appear less than this number of times.
+
+- **word_model**: Specify "SkipGram" to use the Skip-Gram model when producing a distributed representation of words. When enabled, the model uses each word to predict the surrounding window of context words. The skip-gram architecture weighs close context words more heavily than more distant context words. Enabling this option can increase model build time but performs better for infrequently used words. 
```

Comment:
skip-gram is the only approach we support right now. The user doesn't really have an option to change the parameter. It is there for future use and also to make it clear that we use skip-gram.